website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -71,6 +72,48 @@ export const authOptions: AuthOptions = {
   session: {
     strategy: "jwt",
   },
+  callbacks: {
+    async signIn(user, account) {
+      // Check if the user should be marked as an admin.
+      const adminUsers = process.env.ADMIN_USERS.split(",");
```

Comment:
Rather than generating this on every `signIn` call can you do it at the top level and turn this into a map keyed by provider that points to a set of admin IDs?

Something like:
{
 'discord' -> Set('fozziethebeat, andreaskopf'),
 'email' -> Set('fozziethebeat@gmail.com'),
}